config/initializers/secure_headers.rb

```diff
@@ -4,7 +4,7 @@
   config.x_content_type_options = "nosniff"
   config.x_xss_protection = "1; mode=block"
   config.csp = {
-    :enforce     => true,
```

Comment:
@chrisarcand thanks - I was wrong. the migration guide said to `s/enforce/report_only/`